vue/src/Trigger/TriggerEdit.vue

```diff
@@ -95,11 +95,17 @@
           </div>
           <div v-show="trigger.typeMetadata?.hasAdvancedSettings">
             <div class="form-group row multiple">
+              <div class="col s12 input-field m6">
+                <p>
+                  {{ translate('TagManager_TriggerConditionsHelp') }}
+                </p>
+              </div>
+              <div class="col s12 input-field m6 form-help">
+                    <span class="inline-help" v-html="$sanitize(triggerInlineHelpText)">
+                    </span>
+              </div>
```

Comment:
You don't have to, but if you make the following change, it will fix the left padding so that the inline help text lines up with the rest of the help text boxes in the form.
```suggestion
              <div class="col s12 input-field m6">
                <div class="form-help">
                  <span class="inline-help" v-html="$sanitize(triggerInlineHelpText)">
                  </span>
                </div>
              </div>
```

Reply:
@snake14 Updated :+1: